### ISSUE TITLE
Mod seems to not taking Trinkets slots in consideration

### ISSUE THREAD

**Issue by vico93** [Blumbo/KeepInventoryControl#2]
I made a test here and added the atlas item from map atlases (which is equippable on a trinket stot) and a computercraft turtle (a "normal" block") to the whitelist (keepinventory gamerule is false), and did a /kill command. Only the turtle that was in a normal inventory slot was kept, the atlas that was on trinket slot was dropped.

Can this be fixed?

Fabric 1.20.1
KeepInventoryControl 1.0.1

**Comment by Blumbo** [comment]
Are you referring to the trinket slots in [Trinkets mod](https://github.com/emilyploszaj/trinkets)?

**Comment by vico93** [comment]
> Are you referring to the trinket slots in [Trinkets mod](https://github.com/emilyploszaj/trinkets)?

Yep. If a whitelisted item is on a custom slot added by trinkets, it will not respect the rule and drop itself.

**Comment by von-get** [comment]
I have the same issue, with the same mod.  I hope support will be added!
